### ISSUE TITLE
Maxon App (maxonapp) Fails to Install via Jamf with Error Code 16

### ISSUE THREAD

**Issue by jwnola** [Installomator/Installomator#1417]
I am attempting to install the Maxon App (maxonapp) via Jamf Pro. The output of the install is as follows:

```
Executing Policy Install Maxon App
Running script Installomator 10.5...
Script exit code: 16
Script result: 2024-01-12 11:10:31 : REQ   :  : shifting arguments for Jamf
2024-01-12 11:10:31 : REQ   : maxonapp : ################## Start Installomator v. 10.5, date 2023-10-15
2024-01-12 11:10:31 : INFO  : maxonapp : ################## Version: 10.5
2024-01-12 11:10:31 : INFO  : maxonapp : ################## Date: 2023-10-15
2024-01-12 11:10:31 : INFO  : maxonapp : ################## maxonapp
2024-01-12 11:10:31 : INFO  : maxonapp : setting variable from argument NOTIFY=silent
2024-01-12 11:10:31 : INFO  : maxonapp : setting variable from argument DEBUG=0
2024-01-12 11:10:31 : INFO  : maxonapp : BLOCKING_PROCESS_ACTION=tell_user
2024-01-12 11:10:31 : INFO  : maxonapp : NOTIFY=silent
2024-01-12 11:10:31 : INFO  : maxonapp : LOGGING=INFO
2024-01-12 11:10:31 : INFO  : maxonapp : LOGO=/Library/Application Support/JAMF/Jamf.app/Contents/Resources/AppIcon.icns
2024-01-12 11:10:31 : INFO  : maxonapp : Label type: dmg
2024-01-12 11:10:31 : INFO  : maxonapp : archiveName: Maxon.dmg
2024-01-12 11:10:31 : INFO  : maxonapp : no blocking processes defined, using Maxon as default
2024-01-12 11:10:31 : INFO  : maxonapp : name: Maxon, appName: Maxon.app
2024-01-12 11:10:31.975 mdfind[1439:10874] [UserQueryParser] Loading keywords and predicates for locale "en_US"
2024-01-12 11:10:31.975 mdfind[1439:10874] [UserQueryParser] Loading keywords and predicates for locale "en"
2024-01-12 11:10:32.019 mdfind[1439:10874] Couldn't determine the mapping between prefab keywords and predicates.
2024-01-12 11:10:32 : INFO  : maxonapp : App(s) found: /Library/Application Support/Adobe/Keyfiles/AfterEffects/24/CustomHooks/mac/arm64/Maxon Cinema 4D Installer.app
Error running script: return code was 16.
```

I believe that there is confusion with the Cinema 4D version that is installed with Adobe After Effects. Does anyone know a workaround for this? Thanks.

**Comment by meschwartz** [comment]
@bmacphail created [PR 1394](https://github.com/Installomator/Installomator/pull/1394) a few weeks ago to fix this

**Comment by acodega** [comment]
Believe fixed in Installomator 10.6
